scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -220,35 +236,42 @@ case object RedundantBraces extends Rewrite {
   }
 
   /** Some blocks look redundant but aren't */
-  private def retainSingleStatBlock(b: Term.Block): Boolean =
+  private def retainSingleStatBlock(b: Term.Block): Boolean = {
+    val stat = b.stats.head
```

Comment:
Though I know we normally don't pass `Block` whose stats is empty to this function, can we use `headOption` here?

Reply:
let me try. not sure what the function should return if `stats` is empty or more than one.

Reply:
@tanishiking PTAL: https://github.com/scalameta/scalafmt/pull/1635/commits/614d9671895d791c748f6a06f88525d015a1a4a6